test/extended/clusterup.sh

```diff
@@ -158,6 +158,21 @@ function os::test::extended::clusterup::hostdirs () {
     BASE_DIR="${base_dir}" os::test::extended::clusterup::internal::hostdirs
 }
 
+# Tests bringing up the service catalog and provisioning a template
+function os::test::extended::clusterup::service_catalog() {
```

Comment:
yeah, this was what i had in mind.  Just keep in mind that currently *nothing* is running these tests today and there's not even a /test command that will run them right now.  (we need to get one added).